google-cloud-core/src/main/java/com/google/cloud/Policy.java

```diff
@@ -208,27 +275,42 @@ public String etag() {
    * Sets the version of the policy. The default version is 0, meaning only the "owner", "editor",
```

Comment:
I meant at line 275 but folding and the inability to comment on parts that were not changed got me here.... :)